src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/Example6.java

```diff
@@ -0,0 +1,25 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="CommentsIndentation"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+// xdoc section -- start
+public class Example6 {
+  void testMethod() {
+    String breaks = "J"
+        // violation below, 'Comment has incorrect indentation level 4'
```

Comment:
Move violation comment out of target statement - above `String breaks`.

Reply:
Done